staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/types.go

```diff
@@ -27,3 +27,71 @@ type WebhookAdmission struct {
 	// KubeConfigFile is the path to the kubeconfig file.
 	KubeConfigFile string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// EncryptionConfiguration stores the complete configuration for encryption providers.
+type EncryptionConfiguration struct {
```

Comment:
seeing these in webhookadmission/types.go surprises me?

Reply:
Yeah, I share your feeling. I've just put it near the `v1alpha1` group. Where it should be?

Reply:
It's own API group. This has nothing todo with the webhooks.